test/test-trait-cake.js

```diff
@@ -49,32 +49,28 @@ test('cajita-wobbly-point-trait test', t => {
 
 test('hardened-wobbly-point-trait test', t => {
   try {
-    function* AbstractPointLayer(x, y) {
-      const [self] = harden(
-        yield harden({
-          baseGetX() {
-            return x;
-          },
-          getY() {
-            return y;
-          },
-          toString() {
-            return `<${self.getX()},${self.getY()}>`;
-          },
-        }),
-      );
+    function AbstractPointLayer(x, y) {
+      return self => harden({
```

Comment:
Now that we're only testing the hardening cases, you can just remove redundant cases.